google/cloud/bigquery/query.py

```diff
@@ -1310,6 +1310,51 @@ def num_dml_affected_rows(self):
         if num_dml_affected_rows is not None:
             return int(num_dml_affected_rows)
 
+    @property
+    def created(self):
+        """Datetime at which the job was created.
```

Comment:
There might not actually be a "job" associated with the query. Let's use the same description as the REST API docs use.

```suggestion
        """Creation time of this query.
```